### DISCUSSION TITLE
Enable/disable notifications per project

### DISCUSSION THREAD

**Discussion by ubuntudroid** [deszip/BitBot#39] [Feature requests]
It would be great if we could enable and disable notifications per project. Sometimes multiple projects are connected to a bitrise account, but we might only be interested in one or two of them.

Alternatively/additionally completely disabling select project could help clear up the build list.

**Comment by deszip** [comment]
Hi, @ubuntudroid , thanks for suggestions, sounds interesting. 

I'll move this as a feature request to discussions and add to backlog. Hadnt much time recently but plan to get back to new features next month, so this could go to the next release. Stay tuned :)
